src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,55 @@
+.body {
+  background: radial-gradient(
+    circle, rgba(174,238,234,1) 0%, rgba(148,173,233,1) 100%
+  );
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
```

Comment:
Regarding BEM methodology it is not correct to style tags, use class for it.

Reply:
it' class body

Reply:
> 
Sry, it was late tame))